lib/Runtime/Debug/ProbeContainer.h

```diff
@@ -91,7 +91,7 @@ namespace Js
         void Initialize(ScriptContext* pScriptContext);
         void Close();
 
-        WeakDiagStack* GetFramePointers();
+        WeakDiagStack* GetFramePointers(DWORD_PTR dispatchHaltFrameAddress = 0);
```

Comment:
I don't understand the usage of dispatchHaltFrameAddress? when this can be 0? Is this for legacy debugger support?